src/styles/blocks/card.scss

```diff
@@ -0,0 +1,64 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background-color: $background-color-special-white;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  text-align: center;
+  padding-block: 32px 16px;
+
+  transition: all 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__top-title {
+    margin: 0;
+    font-weight: 500;
+    line-height: 18px;
+    font-size: 12px;
+  }
+
+  &:hover .card__top-title {
+    color: #34568b;
+  }
+
+  &__top-image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin-inline: 19px;
+  }
+
+  &__top-text {
+    margin: 40px 16px 16px;
+    text-align: left;
+  }
+
+  &__top-subtitle {
+    margin: 0;
+    margin-top: 4px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $other-text-color;
+  }
+
+  &__review {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0;
+    height: 16px;
+    margin-inline: $margin-inline;
```

Comment:
The variable `$margin-inline` is used but not defined. Please define this variable to ensure the margin is applied correctly.